README.md

```diff
@@ -38,6 +38,17 @@ We also support the aliases of `BREAKING`, `FEATURE`, and `FIX`.
 | 1.2.3            | `#MAJOR#`    | 2.0.0          |
 | 1.2.3            | `#BREAKING#` | 2.0.0          |
 
+### GFM Checklist support
+You may also specify a list of possible scopes in a [GFM checklist](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments)
+ Example:
+ 
+ ### Check the scope of this pr:
+ - [ ] #patch# - bugfix, dependency update
+ - [ ] #minor# - new feature, backwards compatible
+ - [x] #major# - major feature, probably breaking API
+ - [ ] #breaking# - any change that breaks the API
```

Comment:
I'm not sure you need `#breaking` if you already have `#major#` here, and it's probably best to stick with the [official definitions](semver.org):
>Given a version number MAJOR.MINOR.PATCH, increment the:

>MAJOR version when you make incompatible API changes,
>MINOR version when you add functionality in a backwards-compatible manner, and
>PATCH version when you make backwards-compatible bug fixes.